crates/bevy_android/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+
+use once_cell::sync::OnceCell;
+
+pub use ndk::asset::AssetManager;
+pub use winit::platform::android::activity::*;
+
+pub static ANDROID_APP: OnceCell<AndroidApp> = OnceCell::new();
```

Comment:
aah, it'd be a bit of a shame if Bevy goes with a global variable here but I guess that could be ok to start with.

It was intentional to avoid having this `AndroidApp` state be static/global considering that it's possible to start multiple Activities in a single Android application. Instead of making this global there could be some Android-specific trait / API that can be used to associated this `AndroidApp` with the Bevy App, or else perhaps a closure/hook that lets Winit-based applications customize the event loop if necessary.

Here's one example of how I handled this with Egui/Eframe a while back: https://github.com/emilk/egui/pull/1952/commits/6ff1019f0be065cbf54ec2f75d0d89202d46df6f

Another example is Winit itself which has an Android specific builder trait that needs to be used to associate the app with the event loop: https://github.com/rust-windowing/winit/blob/4ed4e918f30c3c465090936b79e7fb561190b4da/src/platform/android.rs#L40